### PR TITLE
CHANGES for 4.9.0

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,141 @@
...
+
+-- 
+
+FusekiModules:
+Issue: https://github.com/apache/jena/issues/1897
+
+There is a change in that the interface for automatically loading 
+modules from the classpath has changed to FusekiAutoModule, The 
+interface FusekiModule is now the configuration lifecycle only. This is 
+to allow for programmatically set up a Fuskei server with Fuseki
```
Comment (kinow) [inline CHANGES.txt:135]: s/Fuskei/Fuseki

File: CHANGES.txt [hunk 2]
```diff
@@ -1,3 +1,141 @@
...
+   Script fix for additional classpath elements
+   https://github.com/apache/jena/pull/1877
+
+-- 
+
+FusekiModules:
+Issue: https://github.com/apache/jena/issues/1897
+
+There is a change in that the interface for automatically loading 
+modules from the classpath has changed to FusekiAutoModule, The
```
Comment (kinow) [inline CHANGES.txt:133]: s/,/. ?

File: CHANGES.txt [hunk 3]
```diff
@@ -1,3 +1,141 @@
...
+   ARQ.symCustomFunctionScriptAllowList
+
+Its value is a comma separated list of function names.
+   "function1,function2"
+Only the functions in this can be called from SPARQL.
+
+As in Jena 4.8.0, the Java system property "jena:scripting" must also be 
+set to "true" to enable script functions.
+   Website (when published):
+    https://jena.apache.org/documentation/query/javascript-functions
```
Comment (kinow) [inline CHANGES.txt:96]: Not sure if it matter, but the indentation is different here, the link has an extra white space before.

File: CHANGES.txt [hunk 4]
```diff
@@ -1,3 +1,141 @@
...
+   https://github.com/apache/jena/pull/1908
+
+There is a new Jena context setting
+   http://jena.apache.org/ARQ#scriptAllowList
+which is on the command line:
+   arq:scriptAllowList
+and java constant
+   ARQ.symCustomFunctionScriptAllowList
+
+Its value is a comma separated list of function names.
```
Comment (kinow) [inline CHANGES.txt:89]: s/Its/It's

Comment (afs) [reply CHANGES.txt:89]: "It's" == "It is" whereas "Its" == "of it".

**Pull request merged**